integrationExamples/gpt/load_pbjs_dfp_concurrently.html

```diff
@@ -5,47 +5,38 @@
 * This is the optimized way of loading prebid.js. It loads both DFP and Prebid concurrently.
 * Publishers may wish to tweak the the MAX_RETRIES  and polling interval ms as desired to mitigate issue https://github.com/prebid/Prebid.js/issues/419
 */
-
+    
 //timeout to control how long bidders have to respond.
 var PREBID_TIMEOUT = 700;
-//how many times to attempt to load DFP if services have not been enabled
-var MAX_RETRIES = 20;
 
 var pbjs = pbjs || {};
 pbjs.que = pbjs.que || [];
-pbjs.retries = 0;
 var googletag = googletag || {};
 googletag.cmd = googletag.cmd || [];
 googletag.cmd.push(function () {
     googletag.pubads().disableInitialLoad();
 });
+
+// Load the Prebid Javascript Library Async. We recommend loading ASAP in page
+loadScript('//acdn.adnxs.com/prebid/not-for-prod/prebid.js');
+//load DFP async.
+loadScript('//www.googletagservices.com/tag/js/gpt.js');
+    
 /* initAdserver will be called either when all bids are back, or
    when the timeout is reached.
 */
 function initAdserver() {
   if (pbjs.initAdserverSet) return;
-
-  //we don't want to call DFP before it is ready
-  if(!googletag.pubadsReady && pbjs.retries <= MAX_RETRIES) {
-    setTimeout(initAdserver, 50); //poll ms can be adjusted as desired.
-    pbjs.retries++;
-    return;
-  }
+  pbjs.initAdserverSet = true;
+    
```

Comment:
Don't drop this check because it helps prevent a race condition where `googletag.pubads().refresh();` is queue before the inline page `googletag.display('div-id');` display call is made. If `.refresh()` executes first, no ads will be fetched (you can test this out easily with a GPT test page and setTimeout).